Testscripts/Windows/Power-Hibernate.ps1

```diff
@@ -180,18 +174,18 @@ install_package "ethtool"
 			foreach ($VMData in $AllVMData) {
 				Copy-RemoteFiles -uploadTo $VMData.PublicIP -port $VMData.SSHPort -files "$constantsFile,$($CurrentTestData.files),$LogDir\*.sh" -username $user -password $password -upload
 				Write-LogInfo "Copied the script files to the VM"
-				Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password -command "bash ./setup.sh" -runAsSudo
+				Run-LinuxCmd -ip $VMData.PublicIP -port $VMData.SSHPort -username $user -password $password -command "bash /home/$user/setup.sh" -runAsSudo
```

Comment:
why need this change? your current user is $user

Reply:
Yes, I like the absolute path clearly.